.github/workflows/release-embedding-sdk.yml

```diff
@@ -51,7 +50,7 @@ jobs:
         id: current-sdk-version
         run: |
           VERSION=$(jq -r .version ./enterprise/frontend/src/embedding-sdk/package.template.json)
-          echo "::set-output name=sdk_current_version::$VERSION"
+          echo "sdk_current_version=$VERSION" >> $GITHUB_OUTPUT
```

Comment:
Remove the soon-to-be deprecated feature. See [the run that contains the error](https://github.com/metabase/metabase/actions/runs/11456987124) here.

![image](https://github.com/user-attachments/assets/ad734cf6-f583-4ecb-a5a4-b8ea18374b65)

And we can verify that the [latest run](https://github.com/metabase/metabase/actions/runs/11457709447) doesn't contain the error anymore.